Hazardous usage of TRUE and FALSE in drivers/nutdrv_qx.h:48

At drivers/nutdrv_qx.h:48 we have:

```
typedef enum { FALSE, TRUE } bool_t;
```

Why not, but...

I tried to compile master on AIX (to rebase balooloo/nut:AIX_packaging_on_master). and the compilation fails:

```
In file included from nutdrv_qx.c:61:
nutdrv_qx.h:48: error: expected identifier before numeric constant
```

My interpretation of that is TRUE or FALSE is already defined to something like 0 or 1 and that makes the enum typedef bad-formed.
Must be confirmed, but I think we have similar problem on Windows.

I suggest to test if TRUE and FALSE are predefined, if not defined bool_t as is, else define bool_t as int.

--- a/drivers/nutdrv_qx.h
+++ b/drivers/nutdrv_qx.h
@@ -45,7 +45,11 @@
 #define DEFAULT_OFFDELAY	"30"	/* Delay before power off, in seconds */
 #define DEFAULT_POLLFREQ	30	/* Polling interval between full updates, in seconds; the driver will do quick polls in the meantime */
 
+#ifndef TRUE
 typedef enum { FALSE, TRUE } bool_t;
+#else
+typedef int bool_t;
+#endif
 
 /* Structure for rw vars */
 typedef struct {

